bin/app.ts

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env node
+/* Copyright OpenSearch Contributors
+SPDX-License-Identifier: Apache-2.0
+
+The OpenSearch Contributors require contributions made to
+this file be licensed under the Apache-2.0 license or a
+compatible open source license. */
+
+import 'source-map-support/register';
+import { App } from 'aws-cdk-lib';
+import { OsClusterEntrypoint } from '../lib/os-cluster-entrypoint';
+
+const app = new App();
+
+new OsClusterEntrypoint(app, {
+  env: { account: process.env.CDK_DEFAULT_ACCOUNT, region: process.env.CDK_DEFAULT_REGION },
+});
```

Comment:
Let the credentials be grabbed from default environment rather than this. Configuring aws credentials should be an independent process

Reply:
This does take whatever default aws configuration that is configured on user's machine. Removing this is resulting in error `Error: Cannot retrieve value from context provider vpc-provider since account/region are not specified at the stack level.`.